index.html

```diff
@@ -0,0 +1,220 @@
+                            <a href="#" class="program-btn-b">SEE THE WHOLE PROGRAM</a>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </section>
+
+        <!-- Feature section -->
+        <section id="feature" class="feature">
+            <div class="container">
+                <div class="feature-content">
+                    <h2 class="section-title">Feature</h2>
+                    <div class="feature-list">
+
+                    </div>
+                    <div class="more-feature">
+                        <a href="#" class="more-btn">See More <i class="fa-solid fa-chevron-down"></i></a>
+                    </div>
+                </div>
+            </div>
+        </section>
+
+        <!-- Partner section -->
+        <section id="partner" class="partner">
+            <div class="container">
+                <div class="partner-content">
+                    <h2 class="section-title partner-title">Partner</h2>
+                    <div class="partner-list">
+                        <div class="partner-item">
+                            <div class="partner-item-img">
+                                <img src="./Images/mozilla.png" alt="">
+                            </div>
+                        </div>
+                        <div class="partner-item">
+                            <div class="partner-item-img">
+                                <img src="./Images/google.png" alt="">
+                            </div>
+                        </div>
+                        <div class="partner-item">
+                            <div class="partner-item-img">
+                                <img src="./Images/naver.png" alt="">
+                            </div>
+                        </div>
+                        <div class="partner-item">
+                            <div class="partner-item-img">
+                                <img src="./Images/kakao.png" alt="">
+                            </div>
+                        </div>
+                        <div class="partner-item">
+                            <div class="partner-item-img">
+                                <img src="./Images/airbnb.png" alt="">
+                            </div>
+                        </div>
+                    </div>
+                    <div class="partner-overlay"></div>
+                </div>
+            </div>
+        </section>
+    </div>
+    <footer class="footer">
+        <div class="container">
+            <div class="footer-content">
+                <div class="footer-logo">
+                    <h1 class="logo">event<span>Work</span></h1>
+                </div>
+                <div class="footer-adress">
+                    <p>2015 Creative Commons Korea <br> Some Rights Reserved</p>
+                </div>
+            </div>
+        </div>
+    </footer>
```

Comment:
- [ ] Kindly update the footer with the right background-color and the positioning of the text as the previous reviewer asked 👍 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/190827067-c6c48599-c917-4024-8e4d-4a00b3bcf24a.png)

## Guidelines:
![image](https://user-images.githubusercontent.com/50721479/190827075-de42e926-fb45-40bb-8117-57122f0afade.png)